docs/features/lambda_dead_letter_queue.md

```diff
@@ -13,33 +13,28 @@ The [workflow-trigger](../workflows/workflow-triggers) for the startSF queue has
 This queue can then be monitored for failures to initiate a workflow.   Please note that workflow failures will not show up in this queue, only repeated failure to trigger a workflow.
 
 ## Named Lambda Dead Letter Queues
-Cumulus provides the ability to configure a default named [Dead Letter Queue](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html) for lambdas.   This is intended to be utilized for non-workflow lambdas (such as ScheduleSF) to capture lambda failures for further processing.
 
-Adding the following configuration to a lambda in `app/lambdas.yml`:
+Cumulus provides configured [Dead Letter Queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html) (`DLQ`) for non-workflow lambdas (such as ScheduleSF) to capture lambda failures for further processing.
```

Comment:
Nit: lambda -> Lambda

Reply:
682e80795 

As an aside, I feel like we should do a doc wide scan/PR on that alone.